dll/win32/msutb/lang/pt-PT.rc

```diff
@@ -0,0 +1,79 @@
+/*
+ * PROJECT:     ReactOS msutb.dll
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Portuguese (Portugal) resource file
+ * TRANSLATOR:  Copyright 2023-2024 Jose Carlos Jesus <zecarlos1957@hotmail.com>
+ */
+
+LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE
+
+STRINGTABLE
+BEGIN
+    IDS_OK               "&OK"
+    IDS_CANCEL           "&Cancelar"
```

Comment:
Normally, OK and Cancel shouldn't require an explicit key accelerator (they would default with ENTER and ESC keys respectively). English lang don't have the accelerator, and I know @JoachimHenze started to remove such redundant ones in shell32 lang files.

Reply:
I will keep that in mind. Thanks.